### PR TITLE
fix: token to access_token and added resultproperty for data retrieval

### PR DESCRIPTION
**Pull request by mspreeuwenberg**
During first implementation we encoutered some minor mistakes.

### CODE REVIEW

File: students.ps1 [hunk 1]
```diff
@@ -145,10 +149,10 @@ try {
             }
             $leerling.Contracts.add($primaryContract)
             # klassen
-            $klassenResult = Invoke-PresentisRestMethod -Uri "$BaseUrl/klassen?schoollocatie=$($schoollocatie.schoollocatieid)&" -Headers  @{Authorization = "Bearer $($responseToken.token)"}
+            $klassenResult = Invoke-PresentisRestMethod -Uri "$BaseUrl/klassen?schoollocatie=$($schoollocatie.schoollocatieid)&" -Headers  @{Authorization = "Bearer $($responseToken.acces_token)"} -ResultProperty "klassen"
```
Comment (JeroenBL) [inline students.ps1]: Shouldn't this be `access_token`? or is it misspelled within Presentis?
also apples to lines: 127, 130, 134, 155 and 177.

And maybe, Also update the _person.ps1_ accordingly with correct values.

File: students.ps1 [hunk 2]
```diff
@@ -96,12 +100,12 @@ try {
 
     if ($config.Environment -eq "Test")
     {
-        $OAuthUrl = https://oauthtest.presentis.nl/oauth2/token
-        $BaseUrl =  https://apitest.presentis.nl/rest/v1
+        $OAuthUrl = "https://oauthtest.presentis.nl/oauth2/token"
```
Comment (JeroenBL) [inline students.ps1:103]: also applies to _persons.ps1_.

**Review by JeroenBL — DISMISSED** [review]

File: students.ps1 (hunk 1 above)
Comment (mspreeuwenberg) [reply students.ps1]: I've checked this once again and "acces_token" is the right one. Furthermore this is applied to the persons.ps1 as well

File: students.ps1 (hunk 2 above)
Comment (mspreeuwenberg) [reply students.ps1:103]: I've applied this fix to persons.ps1 as well

**Review by JeroenBL — APPROVED** [review]

**Pull request merged**